docs/howto/providers/azure.rst

```diff
@@ -0,0 +1,16 @@
+
+Deleting or stopping your virtual machine
+===========================================
+
+After you have finished using your LTJH you might wanto to either Stop or completely delete the Virtual Machine to avoid incurring in subsequent costs. 
```

Comment:
```suggestion
After you have finished using your TLJH you might wanto to either Stop or completely delete the Virtual Machine to avoid incurring in subsequent costs. 
```